common/crypto_utils/avalon_crypto_utils/signature.py

```diff
@@ -85,8 +85,8 @@ def __payload_json_check(self, json_data):
         return param_valid
 
 # -----------------------------------------------------------------------------
-    def __encrypt_workorder_indata(
-            self, input_json_params, session_key, session_iv,
+    def __encrypt_workorder_data(
+            self, input_json_params, data_type, session_key, session_iv,
```

Comment:
Insted of passing data type pass the in/out data array to be encrypted. Please refer 
https://github.com/hyperledger/avalon/blob/master/examples/graphene_apps/python_worker/avalon_worker/work_order_processor.py#L339
and https://github.com/hyperledger/avalon/blob/master/examples/graphene_apps/python_worker/avalon_worker/crypto/worker_encryption.py#L292 and

Reply:
Done